datalad/distribution/publish.py

```diff
@@ -472,12 +472,20 @@ def _get_remote_info(ds_path, ds_remote_info, to, missing):
         track_remote, track_refspec = ds.repo.get_tracking_branch()
         if not track_remote:
             # no tracking remote configured, but let try one more
-            # if we only have one remote, and it has a push target
-            # configured that is "good enough" for us
+            # if we only have one remote, and it has a url
+            # configured that is "good enough" for us. If multiple - we will
+            # clarify in interactive mode
             cand_remotes = [r for r in ds.repo.get_remotes()
-                            if 'remote.{}.push'.format(r) in ds.config]
+                            if 'remote.{}.url'.format(r) in ds.config]
             if len(cand_remotes) > 1:
-                lgr.warning('Target sibling ambiguous, please specific via --to')
+                from datalad.ui import ui
+                if ui.is_interactive:
+                    track_remote = _ui_ask_track_remote(ds, cand_remotes)
+                else:
+                    lgr.warning(
+                        'Target sibling ambiguous (candidates: {}), please '
+                        'specify via --to'.format(', '.join(cand_remotes))
+                    )
```

Comment:
nit: Please don't use `.format` for log messages.

For readability, I find it nicer when a list with an unknown number of items comes at the end of the message.